geode-core/src/test/java/org/apache/geode/test/dunit/cache/internal/JUnit4CacheTestCase.java

```diff
@@ -71,7 +71,7 @@
    * <p>
    * Field is static so it doesn't get serialized with SerializableRunnable inner classes.
    */
-  private static InternalCache cache;
+  protected static InternalCache cache;
```

Comment:
Why has the access been changed? There aren't any direct references to `cache` in this change set.

Reply:
The change I made is in a subclass (`CacheXmlTestCase`) and references `cache` there.

Reply:
OK, I missed that reference to the cache  during my review.